ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  LinkifiedText,
+  Text
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+// TODO: Add color={'gray'} to Text component in buffer-components then use in url below
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  hovered,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  onMouseEnter,
+  onMouseLeave,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
```

Comment:
Hey, @alvarezmelissa87! I'm super interested in this syntax here and wonder what might be happening. 😊 It's more for me learning a new way to include CSS in React components! 😁